src/Forms/VocabForm/VocabForm.jsx

```diff
@@ -113,11 +115,24 @@ const VocabForm = ({
 
   const packageAdded = useCallback(
     (newPackage) => {
-      setSelectedPackage({ value: newPackage.id, label: newPackage.name });
+      setSelectedPackage({
+        value: newPackage.id,
+        label: (
```

Comment:
Maybe you can move this into a separate component so that you can reuse this peace.

Reply:
good idea